### PR TITLE
Wizard component docs

### PR DESCRIPTION
**Pull request by ntacheva**
## New Component Checklist (delete for other types of PRs)

- [x] - Overview article
    - [x] General information on the purpose of the component.
    - [x] Instruction on how to add the component to a razor component.
    - [x] Basic configuration of the component - code snippet with a scenario that is both basic, and meaningful.
    - [x] Provide information on the component's reference.
    - [x] List the available features for the component and/or provide links (slugs) for the articles that explain them in details. For simpler components listing parameters might be enough, for more complex ones sections with a sentence or two that links corresponding articles might be more suitable.
    - [x] Maybe some Example(s) - very common scenarios, notes, typical use case, etc. Not mandatory, depends on the component.

- [x] - Feature/component specific articles - those articles explain the component specific features, which do not suit in any of the before-mentioned articles. Examples of such articles are the [Navigator](https://docs.telerik.com/blazor-ui/components/stockchart/navigator) for the Stock Chart, the [Indeterminate](https://docs.telerik.com/blazor-ui/components/checkbox/indeterminate-state) state for the CheckBox and others. 

- [x] - Events article - applicable if the component expose events. 
Examples of such events could be:

- [x] Update Input Validation, Globalization, Keyboard Navigation, Accessibility Compliance articles as applicable.

- [ ] Link from the Overview components list

### CODE REVIEW

File: components/wizard/overview.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+* [Features](#features)
+
+## Basics
+
+#### To create a basic Telerik Wizard:
+
+1. Use the `TelerikWizard` tag
+1. under its `WizardSteps` tag, set and configure the desired `WizardStep` instances and include the desired content in their `Content` tag
+
+>caption Set up a basic Telerik Wizard. The result from the snippet.
```
Comment (svdimitr) [inline components/wizard/overview.md:32]: The result from the code snippet **below** or remove this sentence.

File: components/wizard/overview.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        [Range(typeof(bool), "true", "true", ErrorMessage = "You must agree with the terms.")]
+        [Display(Name = "Accept Terms and Conditions")]
+        public bool AcceptTerms { get; set; }
+    }
+}
+````
+
+## Features
+
+* `StepperPosition` - `WizardStepperPosition` - Specify where the stepper is rendered against the wizard content. The default is `Top`. Read more about its configuration in the [Layout]() article.
```
Comment (svdimitr) [inline components/wizard/overview.md:139]: missing slug

Maybe paraphrase the second sentence as See the Layout article for more information.

File: components/wizard/structure/content.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+description: Content of the Wizard for Blazor.
+slug: wizard-structure-content
+tags: telerik,blazor,wizard,content
+published: True
+position: 3
+---
+
+# Wizard Content
+
+The `Content` of the wizard is a render fragment of the `WizardStep`. While it is a common use case to integrate a form in the Wizard Content, it can also contain any type of HTML content including various Telerik components as needed to cover your desired scenario.
```
Comment (svdimitr) [inline components/wizard/structure/content.md:13]: I do not understand the first sentence. The Content of the Wizard is a render fragment of the WizardStep. Maybe it should be **The Content is a Child content of a WizardStep**

File: components/wizard/structure/stepper.md [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+description: Stepper of the Wizard for Blazor.
+slug: wizard-structure-stepper
+tags: telerik,blazor,wizard,stepper
+published: True
+position: 0
+---
+
+# Wizard Stepper
+
+One of the main elements of the Wizard component is Stepper. The Wizard component utilizes the [Stepper]({%slug stepper-overview%}) component internally.
```
Comment (svdimitr) [inline components/wizard/structure/stepper.md:13]: is **a**/**the** a stepper. **The Wizard component utilizes the Telerik UI for Blazor Stepper** maybe sounds better.

File: components/wizard/events.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+description: Events of the Wizard for Blazor.
+slug: wizard-events
+tags: telerik,blazor,wizard,events
+published: True
+position: 15
+---
+
+## Events
+
+This article explains the events available in the Telerik DropDownList for Blazor:
```
[the remark this answers is not in the document]
Comment (ntacheva) [reply components/wizard/events.md]: Done

File: components/wizard/events.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+    [CascadingParameter]
+    public DialogFactory Dialog { get; set; }
+}
+````
+
+
+## ValueChanged
+
+The `ValueChanged` event fires after the Step has been changed.
```
[the remark this answers is not in the document]
Comment (ntacheva) [reply components/wizard/events.md:82]: Done

File: components/wizard/events.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+
+This article explains the events available in the Telerik DropDownList for Blazor:
+
+* [OnChange](#onchange)
+* [ValueChanged](#valuechanged)
+* [OnFinish](#onfinish)
+
+## OnChange
+
+The OnChange event is triggered on the current step and fires before the step has changed. It receives an argument of type `WizardStepChangeEventArgs` which exposes the following fields:
```
[the remark this answers is not in the document]
Comment (ntacheva) [reply components/wizard/events.md:21]: Done

File: components/wizard/overview.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+description: Overview of the Wizard for Blazor.
+slug: wizard-overview
+tags: telerik,blazor,wizard,overview
+published: True
+position: 0
+---
+
+# Wizard Overview
+
+The Wizard for Blazor component displays content in sequential, stepwise order. Each step of the Wizard has `Content` (`render fragment`), which can contain any type of HTML content including a [Form]({%slug form-overview%}) component.
```
[the remark this answers is not in the document]
Comment (ntacheva) [reply components/wizard/overview.md:13]: Done

File: components/wizard/overview.md (hunk 1 above)
Comment (ntacheva) [reply components/wizard/overview.md:32]: Done

File: components/wizard/overview.md (hunk 2 above)
Comment (ntacheva) [reply components/wizard/overview.md:139]: Done

File: components/wizard/structure/stepper.md (hunk 4 above)
Comment (ntacheva) [reply components/wizard/structure/stepper.md:13]: Done

File: components/wizard/structure/buttons.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+description: Buttons of the Wizard for Blazor.
+slug: wizard-structure-buttons
+tags: telerik,blazor,wizard,buttons
+published: True
+position: 5
+---
+
+# Wizard Buttons
+
+This article provides details on the buttons available in the Telerik Wizard.
```
[the remark this answers is not in the document]
Comment (ntacheva) [reply components/wizard/structure/buttons.md:13]: Agreed not to change this section.

**Review by svdimitr — APPROVED** [review]

**Pull request merged**